src/components/WwwFrame/Menus/CustomSecondaryMenu.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div>
-		<secondary-menu class="show-for-large" :uses-exact-paths="usesExactPaths">
```

Comment:
This prop was completely unused and was always true.

Reply:
`/ui/src/components/WwwFrame/Menus/DeveloperSecondaryMenu.vue`

Reply:
Sorry @eddieferrer I missed that you had already addressed this file.